docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -53,44 +55,55 @@ You will need the IP addresses of the nodes in the cluster for the next step.
 {{< /tip>}}
 
 
-### Step 2. TPC-C Load Phase
-
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster.
+### 2. Configure the benchmark
 
+If not working with the defaults, we can change the username, password, port, etc. using the configuration file at `config/workload_all.xml`. We can also change the terminals or the physical connections being used by the benchmark using the configuration.
 
 ```sh
-$ ./tpccbenchmark --create=true --load=true \
-  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 \
-  --warehouses=100 \
-  --loaderthreads=48
+<dbtype>postgres</dbtype>
+<driver>org.postgresql.Driver</driver>
+<port>5433</5433>
+<username>yugabyte</username>
+<password></password>
+<isolation>TRANSACTION_REPEATABLE_READ</isolation>
+
+<terminals>100</terminals>
+<numDBConnections>10</numDBConnections>
 ```
 
 {{< note title="Note" >}}
-The default values for `nodes` is `127.0.0.1`, `warehouses` is `10` and
-`loaderthreads` is `10`.
+By default the number of terminals is 10 times the number of warehouses which is the max that the TPC-C spec allows. The number of DB connections is the same as the number of warehouses.
 {{< /note >}}
 
-### Step 3. TPC-C Execute Phase
-
-You can then run the workload against the database as follows:
-
-```sh
-$ ./tpccbenchmark --execute=true \
-  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 \
-  --warehouses=100  \
-  -o outputfile
-```
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li >
+    <a href="#10-wh" class="nav-link active" id="10-wh-tab" data-toggle="tab" role="tab" aria-controls="10-wh" aria-selected="true">
+      10 Warehouses
+    </a>
+  </li>
+  <li>
+    <a href="#100-wh" class="nav-link" id="100-wh-tab" data-toggle="tab" role="tab" aria-controls="100-wh" aria-selected="false">
+      100 Warehouses
+    </a>
+  </li>
+  <li>
+    <a href="#1000-wh" class="nav-link" id="docker-tab" data-toggle="tab" role="tab" aria-controls="docker" aria-selected="false">
+      1000 Warehouses
+    </a>
+  </li>
+</ul>
 
-{{< tip title="Tip" >}}
-You can also load and run the benchmark in a single step:
-```sh
-$ ./tpccbenchmark --create=true --load=true --execute=true \
-  --nodes=127.0.0.1,127.0.0.2,127.0.0.3 \
-  --warehouses=100 \
-  -o outputfile
-```
-{{< /tip>}}
+<div class="tab-content">
+  <div id="10-wh" class="tab-pane fade show active" role="tabpanel" aria-labelledby="10-wh-tab">
+    {{% includeMarkdown "10-wh/tpcc-ysql.md" /%}}
+  </div>
+  <div id="100-wh" class="tab-pane fade" role="tabpanel" aria-labelledby="100-wh-tab">
+    {{% includeMarkdown "100-wh/tpcc-ysql.md" /%}}
+  </div>
+  <div id="1000-wh" class="tab-pane fade" role="tabpanel" aria-labelledby="1000-wh-tab">
+    {{% includeMarkdown "1000-wh/tpcc-ysql.md" /%}}
+  </div>
+</div>
 
 ## TPC-C Benchmark Results
```

Comment:
Change this section title to `4. Benchmark results`

Reply:
Also, ideally the benchmark results should be added for each warehouse number.

Reply:
I dont have the result text for all the 3 workloads with 30 mins saved.
Will get it done the next time I run the workloads.

Reply:
Done